src/constants/index.ts

```diff
@@ -215,4 +215,4 @@ export const BLOCKED_PRICE_IMPACT_NON_EXPERT: Percent = new Percent(JSBI.BigInt(
 export const MIN_ETH: JSBI = JSBI.exponentiate(JSBI.BigInt(10), JSBI.BigInt(16)) // .01 ETH
 export const BETTER_TRADE_LINK_THRESHOLD = new Percent(JSBI.BigInt(75), JSBI.BigInt(10000))
 
-export const UNSUPPORTED_BRIDGE_TOKENS = ['FUSE', 'WFUSE']
+export const UNSUPPORTED_BRIDGE_TOKENS = ['FUSE']
```

Comment:
@mul53 WFUSE is still not supported on the bridge.
Should be allowed only on swapping

Reply:
Is creating two separate arrays fine one for unsupported bridge tokens and another for unsupported swap tokens?

Reply:
Hey, the unsupported array only relevant for the bridge. Because relaying this tokens will cause unexpected behavior.
No contstrains for swapping tokens.

Reply:
okay closing this as the array is only used by the bridge